tests/drivers/test_tracing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# Copyright 2018 Criteo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+import mock
+
+from opencensus.trace import tracer
+from opencensus.trace import execution_context
+
+from biggraphite.drivers import tracing
+
+
+class TestTracingTrace(unittest.TestCase):
+    """Test tracing integration of Biggraphite."""
+
+    def testDecoratorTrace(self):
```

Comment:
Python function are usually `lower_snake_camel_case`

I'd also prefer the function name to explain what the test does (in a test run log, we have the function name but not the docstring). Example (feel free to find a better one): 

    test_decorating_with_tracing_function_should_create_a_span_with_same_name_as_foo